lib/tasks/clean/schema.rake

```diff
@@ -15,7 +15,7 @@ namespace :clean do
     schema_hash = {}
     schema_yaml = "---\n"
     Ekylibre::Record::Base.connection.tables.sort.delete_if do |table|
-      %w[schema_migrations spatial_ref_sys oauth_access_grants oauth_access_tokens oauth_applications].include?(table.to_s)
+      %w[schema_migrations spatial_ref_sys oauth_access_grants oauth_access_tokens oauth_applications target_distributions].include?(table.to_s)
```

Comment:
Table is still here and not a special table, so we don't need to exclude it from clean process